components/function-controller/Makefile

```diff
@@ -127,3 +125,8 @@ export GOBIN := $(GOPATH)/src/$(PKG)/bin
 .PHONY: bin/kustomize
 bin/kustomize:
 	@go install ./vendor/sigs.k8s.io/kustomize
+
+.PHONY: ko
+ko:
+	@go get github.com/google/ko/cmd/ko
```

Comment:
Don't forget `-u` to ensure ko is up-to-date.

Reply:
good point, fixed